nbsdecoder.d.ts

```diff
@@ -73,6 +81,13 @@ export declare class NbsDecoder {
    */
   public constructor(paths: string[]);
 
+  /**
+   * The index data of this decoder.
+   *
+   * Contains the a list of packet timestamps for each typeSubType in the loaded nbs files.
+   */
+  public readonly index: TypeIndex[];
```

Comment:
This feels like it should be a Map<NbsTypeSubtype, NbsTimestamp[]>
or it should be a function that takes a type_subtype and returns a set of timestamps?

Reply:
Changed to a function